ray_on_golem/cli.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+import click
+
+
+def with_datadir(cli_func):
+    from ray_on_golem.server.settings import DEFAULT_DATADIR
+
+    return click.option(
+        "--datadir",
+        type=Path,
+        help=f"Ray on Golem's data directory. By default, uses a system data directory: {DEFAULT_DATADIR}",
+    )(cli_func)
```

Comment:
I am not convinced `with_datadir` is a good name for this function.

Reply:
well, that's hardly an applicable comment ;) ... and I believe the original is not bad -> it's a decorator which tells the command that it accepts the datadir option... if you don't think this is a good name, please suggest a better one, thanks :)